src/rgw/rgw_rest_s3.cc

```diff
@@ -463,6 +463,10 @@ int RGWGetObj_ObjStore_S3::send_response_data(bufferlist& bl, off_t bl_ofs,
       }
     } catch (const buffer::error&) {} // omit x-rgw-replicated-from headers
   }
+  if (auto i = attrs.find(RGW_ATTR_OBJ_REPLICATION_TIMESTAMP);
+      i != attrs.end()) {
+    dump_header(s, "x-amz-replicated-at", i->second);
```

Comment:
this doesn't exist in aws, does it? we should use the x-rgw- prefix for rgw extensions